SwiftGit2/SwiftGit2.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  SwiftGit2.swift
+//  SwiftGit2
+//
+//  Created by Andrew Breckenridge on 10/23/17.
+//  MIT.
+//
+
+import Clibgit2
+
+// swiftlint:disable:next identifier_name
+let SwiftGit2Init: () = {
```

Comment:
Suggest turning this to a private variable and wrapping with a function. Please refer to my previous comment.

Reply:
Addressed in commit https://github.com/SwiftGit2/SwiftGit2/pull/138/commits/5a28aea51a610f3e2222bfe0724e76bbd8018b07